doc/sphinx/user/intro.md

```diff
@@ -56,6 +56,19 @@ For what exactly to cite and suggestions for acknowledgments, please see **<http
 
 Also see {cite:t}`aspectmanual,aspect-doi-v1.5.0,aspect-doi-v2.0.0,aspect-doi-v2.0.1,kronbichler:etal:2012,heister:etal:2017`.
 
+## Publishing with ASPECT
+Open research statements are now a common requirement when publishing research. These support reuse, validation, and citation and often take the form of *Data availability, Data access, Code availability, Open Research*, and *Software availability* statements.
+ We recommend depositing input files that allow your published research to be reproduced and output model data in support of your research outcomes and figures. In addition, consider depositing model files that may be reused by others.
+
+
+Remember to cite software and data in your text as well as in your Data Availability or similar statement.
+
+File should be deposited in an approved repository.
```

Comment:
This sentence belongs to the end of the first paragraph you add. But it would still be useful to explain what "approved repository" is supposed to be.

Reply:
This statement is a follow on to you comment on line 61 .  Line 70 is the url to the CIG website which has more explicit wording.  I will modify to make the links between the statements stronger.